src/main/java/gov/cabinetoffice/gap/adminbackend/services/UserService.java

```diff
@@ -29,4 +42,17 @@ public void migrateUser(final String oneLoginSub, final UUID colaSub) {
         });
     }
 
+    public void verifyAdminRoles(final String emailAddress, final String roles) {
+        final String url = userServiceConfig.getDomain() + "/v2/validateAdminSession";
+        final HttpHeaders requestHeaders = new HttpHeaders();
+        requestHeaders.add("emailAddress", emailAddress);
```

Comment:
I think these should be passed in as query params, any reason why you're passing them in as headers?

Reply:
No good reason, using query params now